yunwoo/programmers-2/다음-큰-숫자.js

```diff
@@ -0,0 +1,22 @@
+function solution(n) {
+  const binaryN = n.toString(2).replace(/0/g, '');
+  let nextBigNumber = n;
+
+  while (true) {
+    nextBigNumber++;
+    const binaryNextBigNumber = nextBigNumber.toString(2).replace(/0/g, '');
+
+    if (binaryNextBigNumber === binaryN) {
+      break;
+    }
+  }
+
+  return nextBigNumber;
+}
```

Comment:
```suggestion
function solution(n) {
  const binaryN = n.toString(2).replace(/0/g, '');
  let nextBigNumber = n;

  while (true) {
    const binaryNextBigNumber = (++nextBigNumber).toString(2).replace(/0/g, '');

    if (binaryNextBigNumber === binaryN) {
      return nextBigNumber;
    }
  }
}
```
- 풀이잘봤습니다!! 🙇‍♂️
- break하고 while문 밖에서 return도 좋고, 또 하나의 방법으로 break에다 그냥 return문을 쓰셔도 좋습니다!

Reply:
- 아하 while문 안에서 바로 리턴을 했어도 되네요!! 감사합니다🙇‍♂️